packages/graphback-core/src/graphql/annotations.ts

```diff
@@ -33,4 +33,10 @@ export const parseFieldAnnotations = (node: FieldDefinitionNode): InputModelFiel
     }
 
     return fieldAnnotations;
+}
+
+
+export const parseTypeAnnotations = (node: TypeDefinitionNode): InputModelFieldAnnotations => {
```

Comment:
At the minute it looks like the parsed annotations would not correctly implement the`InputModelFieldAnnotations` interface. Should this return type be removed altogether?

```suggestion
export const parseTypeAnnotations = (node: TypeDefinitionNode) => {
```